eng/targets/Cpp.Common.props

```diff
@@ -9,6 +9,12 @@
     <IntDir>$(IntermediateOutputPath)</IntDir>
   </PropertyGroup>
 
+  <!-- These test projects are skipped because you cannot test native ARM64 projects on a x86/x64 machine. -->
+  <PropertyGroup Condition=" '$(Platform)' == 'ARM64' AND '$(TargetArchitecture)' == 'x64' ">
+    <SkipNativeTest>true</SkipNativeTest>
+    <TestProjectSkipReason>You cannot test native ARM64 projects on a x86/x64 machine</TestProjectSkipReason>
+  </PropertyGroup>
```

Comment:
If this change and the one in eng/targets/Cpp.Common.targets only affect local / incorrect test attempts, please move them to the "dev experience" PR

Reply:
Nope, without this the builds fail, as the test job will try to run the arm64 tests